.github/workflows/docs.yml

```diff
@@ -0,0 +1,38 @@
+name: Docs
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  publish_docs:
+    name: Publish docs
+    runs-on: ubuntu-latest
+    #if: github.event_name == 'release' && github.event.action == 'published'
```

Comment:
I'm not sure whether we would want this to be like this forever, but at least while sprinting in PyCon this is super useful.